src/main/java/io/github/btmxh/apartmentapp/User.java

```diff
@@ -0,0 +1,26 @@
+package io.github.btmxh.apartmentapp;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+public class User {
+    private static User instance;
+    private String username;
+
+    private User() {}
+
+    public static User getInstance() {
```

Comment:
Nên tránh dùng singleton (chỉ dùng cho mấy cái như DB connection) thôi. Ở đây để truyền user từ trang login sang page thì ô có thể dùng hàm getController() của FXMLLoader rồi tạo hàm setUser trong controller. Với lại để cho dễ code property ô nên cho User là một immutable class (ko thì có thể dùng javafx [beans](https://dzone.com/articles/the-bean-class-for-javafx-programming) n mà nó khó code hơn), khi đấy thì khi initialize ô bind username vào cái username label.

Reply:
dùng beans thì ô tạo field User trong file Controller, còn dùng kiểu immutable thí tạo field ObjectProperty<User>